pkg/setting/setting.go

```diff
@@ -1590,7 +1590,7 @@ func readUserSettings(iniFile *ini.File, cfg *Cfg) error {
 	cfg.AutoAssignOrgRole = users.Key("auto_assign_org_role").In("Editor", []string{"Editor", "Admin", "Viewer"})
 	VerifyEmailEnabled = users.Key("verify_email_enabled").MustBool(false)
 
-	cfg.CaseInsensitiveLogin = users.Key("case_insensitive_login").MustBool(false)
+	cfg.CaseInsensitiveLogin = users.Key("case_insensitive_login").MustBool(true)
```

Comment:
I don't think that this configuration [is documented](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#users) anywhere: is it? I think it should

Reply:
@papagian we want this setting to be removed and not used by the user. So we should not document it. i guess we could add some note to the code that it is internal